src/integration/pages/articles/news/__snapshots__/canonical.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Canonical Articles A11y I can see the skip to content link 1`] = `"Skip to content"`;
 
-exports[`Canonical Articles Analytics ATI 1`] = `"<img height="1px" width="1px" alt="" style="position:absolute" src="https://logws1363.ati-host.net?s=598286&amp;s2=64&amp;p=news.articles.c0g992jmmkko.page&amp;x1=[urn%3Abbc%3Aoptimo%3Aasset%3Ac0g992jmmkko]&amp;x2=[responsive]&amp;x3=[news]&amp;x4=[en-gb]&amp;x7=[article]&amp;x8=[simorgh-nojs]&amp;x9=[Royal%2520wedding%2520flowers%2520given%2520to%2520Hackney%2520hospice]&amp;x11=[2019-05-28T13%3A38%3A19.412Z]&amp;x12=[2019-05-28T13%3A38%3A19.412Z]&amp;x13=[Wedding%2Bof%2BPrince%2BHarry%2Band%2BMeghan%2BMarkle~Duchess%2Bof%2BSussex~Hackney~Windsor]&amp;x14=[2351f2b2-ce36-4f44-996d-c3c4f7f90eaa~803eaeb9-c0c3-4f1b-9a66-90efac3df2dc~dba7dfeb-87d3-4801-80ba-fa8be6c9555e~f94940f0-476c-4ec7-b48d-e58120bb1956]&amp;x17=[Wedding%2Bof%2BPrince%2BHarry%2Band%2BMeghan%2BMarkle~Duchess%2Bof%2BSussex~Hackney~Windsor]"/>"`;
+exports[`Canonical Articles Analytics ATI 1`] = `"<img height="1px" width="1px" alt="" style="position:absolute" src="https://logws1363.ati-host.net?s=598288&amp;s2=64&amp;p=news.articles.c0g992jmmkko.page&amp;x1=[urn%3Abbc%3Aoptimo%3Aasset%3Ac0g992jmmkko]&amp;x2=[responsive]&amp;x3=[news]&amp;x4=[en-gb]&amp;x7=[article]&amp;x8=[simorgh-nojs]&amp;x9=[Royal%2520wedding%2520flowers%2520given%2520to%2520Hackney%2520hospice]&amp;x11=[2019-05-28T13%3A38%3A19.412Z]&amp;x12=[2019-05-28T13%3A38%3A19.412Z]&amp;x13=[Wedding%2Bof%2BPrince%2BHarry%2Band%2BMeghan%2BMarkle~Duchess%2Bof%2BSussex~Hackney~Windsor]&amp;x14=[2351f2b2-ce36-4f44-996d-c3c4f7f90eaa~803eaeb9-c0c3-4f1b-9a66-90efac3df2dc~dba7dfeb-87d3-4801-80ba-fa8be6c9555e~f94940f0-476c-4ec7-b48d-e58120bb1956]&amp;x17=[Wedding%2Bof%2BPrince%2BHarry%2Band%2BMeghan%2BMarkle~Duchess%2Bof%2BSussex~Hackney~Windsor]"/>"`;
```

Comment:
I was wondering why this has changed? Did we expect it to?

Reply:
Looks like this changed from [NEWS_PS_TEST](https://github.com/bbc/simorgh/blob/ee2a250284a32bdd9eb38f43232aecdf36ad5879/src/app/lib/analyticsUtils/index.js#L21) to [NEWS_GNL_TEST](https://github.com/bbc/simorgh/blob/ee2a250284a32bdd9eb38f43232aecdf36ad5879/src/app/lib/analyticsUtils/index.js#L24), needs a bit more digging but looks like the change might've affected the logic somewhere in that file

Reply:
It's coming from here: https://github.com/bbc/simorgh/pull/11041/files#diff-cf9f800225e789383232d6d4a4cf51f2e93765040e29d4c9d78dae8ab596fda4R104

Reply:
Before going live with a change like this we need to do some testing to make sure the correct values are being sent to ATI still.

Is this caused by this line here: https://github.com/bbc/simorgh/pull/11041/files#diff-3d175f84c5afab639fff2384ebf1f56d05c80b62d721c5e4b223f6cd1beadc26R25  ?

Reply:
Thanks for pointing this out. So does this mean that when our audience are viewing amp news articles in the UK, then analytics are going to the right destination? (And can we confirm somehow that NEWS_PS will be used on live - you should be able to spin up Simorgh in live mode, and check what the ATI value is when you set the headers accordingly - `yarn build:live && yarn start`)

(Let me know if you need a hand with this)